2005-09-15  Tom Tromey  <tromey@redhat.com>

	Workaround for PR classpath/23863:
	* java/lang/mprec.h (MAX_BIGNUM_WDS): Define as 128 on
	non-Pack_32 platforms.

--- libjava/java/lang/mprec.h	2004/04/09 14:04:48	1.4
+++ libjava/java/lang/mprec.h	2005/09/15 17:10:29	1.5
@@ -291,7 +291,12 @@
 
 
 #define MAX_BIGNUMS 16
+#ifdef Pack_32
 #define MAX_BIGNUM_WDS 32
+#else
+  /* Note that this is a workaround for */
+#define MAX_BIGNUM_WDS 128
+#endif
 
 struct _Jv_Bigint
 {
